apis/apiextensions/v1/composition_types.go

```diff
@@ -607,7 +709,7 @@ const (
 	ConnectionDetailTypeUnknown                 ConnectionDetailType = "Unknown"
 	ConnectionDetailTypeFromConnectionSecretKey ConnectionDetailType = "FromConnectionSecretKey"
 	ConnectionDetailTypeFromFieldPath           ConnectionDetailType = "FromFieldPath"
-	ConnectionDetailTypeFromValue               ConnectionDetailType = "FromValue"
+	ConnectionDetailTypeFromValue               ConnectionDetailType = "From Value"
```

Comment:
This seems like an unintentional change.

Reply:
yes it was a inadvertent space.  I've pushed a fix in 21a616de.